### PR TITLE
fix version streams for loki and pass dependencies.provides to other packages

### PR DESCRIPTION
**Pull request by Dentrax**
* rename `loki-3.0` to `loki-3`
* add missing `dependencies.provides` for some packages
* fixes `loki-3` build due to missing config file

### CODE REVIEW

File: gtk-2.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package:
-  name: gtk-2.0
+  name: gtk-2
```
Comment (xnox) [inline gtk-2.yaml:2]: @Dentrax gtk-2.0 is not a version stream, but an obsolete fork of the project. All other distributions use "2.0" for the ancient fork, and "3" for the currentish fork. Thus my preference would be to leave this historical name as is.

Note, all subpackages are still called "2.0" - did you want to change all of those too? and thus all reverse depends?

Comment (Dentrax) [reply gtk-2.yaml:2]: Ah TIL that, thank you! FWIW better keep this as `-2.0`.

**Review by rawlingsj — CHANGES_REQUESTED** [review]
we'll need to bump the epochs so we get the updated package metadata for the provides

**Comment by Dentrax** [comment]
> we'll need to bump the epochs so we get the updated package metadata for the provides

In this case pipeline would take ~1-2 hours. Actually this is not a blocker thing but bot eventually bump the versions so that we get the updated package metadata right? @rawlingsj 

But if you say we should do it in this PR, happy to bump epochs!

**Comment by rawlingsj** [comment]
Yeah I think it's best in case anything breaks we find out now rather than later down the line.  Maybe bump the epochs and I can merge when I wake while it is quiet?

**Comment by Dentrax** [comment]
@rawlingsj this is now ready to go!

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**